docs/developer/events.md

```diff
@@ -41,12 +41,17 @@ public class ExampleEventSubscriber implements EventSubscriber {
 }
 ```
 
-All the events are subclass of the `Event` abstract class so to filter events it's enough to check the type at runtime.
+To filter events, the payload classes of the events can be used. There are 5 "Intermediate superclasses" (AssetEventPayload, 
+ContractDefinitionEventPayload, ContractNegotiationEventPayload, PolicyDefinitionEventPayload, TransferProcessEventPayload) of 
+which one payload class inherits in each case. These intermediate superclasses inherit again from the class EventPayload. Thus, 
+in addition to filtering on a specific event, such as TransferProcessCompleted, it is possible to react to a group of events 
+that generally have to do with Assets, ContractDefinition, ContractNegotiation, PolicyDefinition and TransferProcess.
+
 ```java
 public class ExampleEventSubscriber implements EventSubscriber {
     
     public void on(Event event) {
-        if (event instanceof AssertCreated) {
+        if (event.getPayload() instanceof TransferProcessEventPayload) {
             // react only to AssertCreated events
```

Comment:
the comment should be updated as well

Reply:
Updated it. Sorry, overread this comment.